.github/workflows/docker-build-stackup-bundler.yml

```
@@ -0,0 +1,76 @@
+name: Build stackup-bundler docker image
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository_owner }}/mothership-l2launcher-stackup-bundler
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      packages: write
+      id-token: write
+
+    steps:
+      - uses: actions/checkout@v3
+      
+      - id: stackup-bundler-changes
+        uses: tj-actions/changed-files@v35
+        with:
+          files: "stackup-bundler"
+
+      - name: Install cosign
+        if: github.event_name != 'pull_request' && steps.stackup-bundler-changes.outputs.any_changed == 'true' && github.repository_owner == 'planetarium'
+        uses: sigstore/cosign-installer@v3.1.1
+
+      - name: Set up Docker Buildx
+        if: steps.stackup-bundler-changes.outputs.any_changed == 'true'
+        uses: docker/setup-buildx-action@v3.0.0
+
+      - name: Log into registry ${{ env.REGISTRY }}
+        if: github.event_name != 'pull_request' && steps.stackup-bundler-changes.outputs.any_changed == 'true' && github.repository_owner == 'planetarium'
+        uses: docker/login-action@v3.0.0
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Extract Docker metadata
+        id: meta
+        if: steps.stackup-bundler-changes.outputs.any_changed == 'true'
+        uses: docker/metadata-action@v5.0.0
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+          tags: |
+            type=raw,value=latest,enable={{is_default_branch}}
+            type=ref,event=branch
+            type=ref,event=pr
+            type=sha
+
+      - name: Build and push Docker image
+        id: build-and-push
+        if: steps.stackup-bundler-changes.outputs.any_changed == 'true'
+        uses: docker/build-push-action@v5.0.0
+        with:
+          context: "{{defaultContext}}:stackup-bundler"
```

We may use https://github.com/stackup-wallet/stackup-bundler/blob/main/Dockerfile here,

```suggestion
          context: "https://github.com/stackup-wallet/stackup-bundler.git"
```